.storybook/main.js

```diff
@@ -0,0 +1,51 @@
+const path = require("path");
+const { storyTree } = require("./storyTree");
+
+const stories = ["./welcome.story.js"];
+storyTree.forEach(catList => {
+  catList.components.forEach(component => {
+    const path = `../packages/${component}/stories/${component}`;
+    stories.push(
+      `${path}.stories.mdx`,
+      `${path}.stories.js`,
+      `${path}.examples.stories.js`,
+      `${path}.backyard.stories.js`,
+      `${path}.tests.stories.js`
+    );
+  });
+});
```

Comment:
This is how the order of the stories is controlled.  Going forward we should adhere to a consistent structure of `stories` files so that we can get the stories within a component ordered the way we want.

Reply:
I think this will be more readable, but just my own opinion
```jsx
const path = `../packages/${component}/stories`;
    stories.push(
      `${path}/${component}.stories.mdx`,
      `${path}/${component}.stories.js`,
      `${path}/${component}.examples.stories.js`,
      `${path}/${component}.backyard.stories.js`,
      `${path}/${component}.tests.stories.js`
    );
```